Islandora/islandora
# added custom module dependencies enforced section under install/config for #1707

**Related issue**: (link)
https://github.com/Islandora/documentation/issues/1707

# What does this Pull Request do?
makes it so that the islandora_breadcrumbs module will clean up the configuration related to the module when it is uninstalled.

A brief description of what the intended result of the PR will be and/or what problem it solves.
When the drush command to install / uninstall this module was executed, there would be an error upon installing if the configuration remained behind from a previous install. That error would be as follows:
```
$ drush pm-enable islandora_breadcrumbs

In PreExistingConfigException.php line 65:
                                                                                                                         
  Configuration objects (islandora.breadcrumbs) provided by islandora_breadcrumbs already exist in active configuration  
```

# What's new?
A in-depth description of the changes made by this PR. Technical details and possible side effects.

* the config/install yml file was updated to enforce the module has install "islandora.breadcrumbs" settings to uninstall when removed.

# How should this be tested?

* check out this code branch
* from the command prompt 
  * run `cd /var/www/html/drupal/web `
  * run `drush pm-uninstall islandora_breadcrumbs`
  * run `drush pm-enable islandora_breadcrumbs` (this step would have displayed the error before the code change)

You should see no errors. If you do no see any errors, the code change worked to remove the configuration.
